music_assistant/server/providers/opensubsonic/sonic_provider.py

```diff
@@ -495,7 +495,11 @@ async def get_album(self, prov_album_id: str) -> Album:
         """Return the requested Album."""
         try:
             sonic_album: SonicAlbum = await self._run_async(self._conn.getAlbum, prov_album_id)
-            sonic_info = await self._run_async(self._conn.getAlbumInfo2, aid=prov_album_id)
+            try:
+                sonic_info = await self._run_async(self._conn.getAlbumInfo2, aid=prov_album_id)
+            except KeyError:
```

Comment:
The 'albumInfo' field in the response from the server is required (see: https://opensubsonic.netlify.app/docs/endpoints/getalbuminfo2/). If the server does not implement that endpoint it should return an error saying so.

On top of accommodating it here, please open a bug with the Nextcloud devs (if they claim to support Open Subsonic) about the missing item in the response.

Reply:
I can confirm MASS works without the workaround patch after monkey-patching my Nextcloud Music.

It seems they return `albumInfo2` instead of `albumInfo`.

https://github.com/owncloud/music/blob/master/lib/Controller/SubsonicController.php#L1524
https://github.com/owncloud/music/blob/master/lib/Controller/SubsonicController.php#L327

Also opened issue on the Owncloud/Nextcloud Music repository.

https://github.com/owncloud/music/issues/1123

Reply:
Reverting b952f26f9c0502167fdaebe4d434066e430ec424